### PR TITLE
Added HTML helpers, and some constants

### PR DESCRIPTION
**Pull request by fadyham**

### CODE REVIEW

File: PHPfiles/html_helpers.php [hunk 1]
```diff
@@ -1,14 +0,0 @@
-<?php
```
Comment (saraguirgis) [inline PHPfiles/html_helpers.php]: Why is this file in red and the others in green?

File: PHPfiles/Constants.php [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * A class to hold Theme related constants.
+ */
+class ThemeConstants {
+    /** Table header background color. */
+    const TableHeaderBGColor = "#D9EDF7";
+    /** Table cell background color when there's no lunch available. */
+    const TableCellNoLunchBGColor = "#D3D3D3";
+    /** Table cell background color when lunch is available to order. */
```
Comment (saraguirgis) [inline PHPfiles/Constants.php]: Should this say "when lunch is NOT available to order"?

File: PHPfiles/Constants.php [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * A class to hold Theme related constants.
+ */
+class ThemeConstants {
+    /** Table header background color. */
+    const TableHeaderBGColor = "#D9EDF7";
```
Comment (saraguirgis) [inline PHPfiles/Constants.php:7]: what does "const" mean?  would you call this constant like this, " ThemeConstants::TableHeaderBGColor " ?

File: PHPfiles/Constants.php (hunk 2 above)
Comment (fadyham) [reply PHPfiles/Constants.php]: Ranamed

File: PHPfiles/Constants.php (hunk 3 above)
Comment (fadyham) [reply PHPfiles/Constants.php:7]: Yes.

**Review by saraguirgis — APPROVED** [review]

**Pull request merged**